src/taggle.js

```diff
@@ -302,9 +308,11 @@
     Taggle.prototype._attachEvents = function() {
         var self = this;
 
-        _on(this.container, 'click', function() {
-            self.input.focus();
-        });
+        if (this.settings.focusInputOnContainerClick) {
+            _on(this.container, 'click', function() {
```

Comment:
Maybe I'm styling things a bit differently. I opted to place the selected items outside of the input and I was observing some unwanted input focusing behavior associated with that click handler. Tomorrow I can delve a bit deeper if you're interested.



> On Jan 31, 2016, at 6:17 PM, Sean Coker <notifications@github.com> wrote:
> 
> In src/taggle.js:
> 
> > @@ -302,9 +308,11 @@
> >      Taggle.prototype._attachEvents = function() {
> >          var self = this;
> >  
> > -        _on(this.container, 'click', function() {
> > -            self.input.focus();
> > -        });
> > +        if (this.settings.focusInputOnContainerClick) {
> > +            _on(this.container, 'click', function() {
> Curious about this setting. When would you expect to click a "textarea" and not have it focus?
> 
> —
> Reply to this email directly or view it on GitHub.
>